topics/metagenomics/tutorials/plasmid-metagenomics-nanopore/tutorial.md

```diff
@@ -0,0 +1,514 @@
TTTCTAAGGAGAAAATGAACAAAAGCTATCACTAAATCGTATTTCCAATAGAGGTTTCGTCCTCTTTGAAAGGGGTAGTTCATGAAAATTAGTTCATGGAAATGTTTATGATTTAGGTTGGAGTTATTAGGCGCAAAGAAGAAATATAAAAGGATAATTGAAGTTCTAAATCATTATGGTGATTGAGAGGCGTTATGAGCACAGCTAAGGAAAGTTAAGGAAAAAGGCACTCCAATGAGTAGTGCCAATGAATATTCTGGTCGTGATAGTATGAATCAAGATTACATCTAGGCAATCCATGGATATTATTATAGGTTTCTGTGGGCTGTTGGTTCGGTATTAAGTCAATTGTAGATGTAACCGAAGAAGAACTGAAGATTTAGGATACGAAGTATATAGGATAAGAGTTAGCGATCTTATCAGAAACTTTTTCTGAGTGGTGGGGATACATCTACTCCATTCAAAAGATATAATTCCTTGCGGGATTTTGGTGATTCCTTAAGATGTAGAGTACTACCCACACATACTAGCTGAAGCTGTAATTCATGAAATAAAGATAGAAAAATCTAGAAGGAAAATCAAAGGTTAAAGAAACGACTTTCCTTATGAATCAACTAAAAACATAAGATGAAGTGAGTTGCCCAAGTGGTCTAACATAATTTCTATTTATTAGGTGATAGGGCGAAACTGAAAGAAAAGAAATCTTCGAGATAGGGGTTTAAGCAAATCTGAAGTTGATATAATCATCCATCATGACAGGAGTCTGAGCACCCTTATGGCCAGCAAACAGCAAAAGCTATACTTGATGCTGACTATTTTGTTAAAATAATCAGGCCAGAGCTCACCTTAAATCTAAGTTAAACGCTTTCTTGGTCTATTCATGGGCTTAATGGCCTTACGCCAAACGCGCATGAGGAAGACATGTACGCAGCTCATTCTGCCTCTTTACAATCTGCATGTCTTTCCAGACAGGTAGGTGCTGCTATTTTAGATATAAAGGAAATTTAATTGCTGTTGGGCGCAATGATGTTCTAAATTTAACAGTGTGGCCTTTATAGTGCTGATGATGGTGAATGATCATCGCTGTGTCTACAAAGGTGGCAAATGTTACAATGAAGCAAGAAAATTAAATAAAGATAAATACAGCAAATATTGCTTTTCTGATAGGTTAAGAATTTGTCTGTTCAGTTAACTCAAGAACAAGCAGAAAAGGTTGCTGGAATGTCTACGAAGGTACTCCTGTATCATCAATCGACAGTACTCGCTCTATTCATGCTGAAATGGATGCTTCACAGCCCATCTTGCTAGACTCGAGATAGTGGTTTTGAAGATAAGGTTTTATATACAACTACGTTCCCCTGTCATAATTGTGCAGGGTATATTGTTGCTGTTGGTATCAATAAAGTTGTTTATTGAACATATGAAAAAGCAGCTTTAGAGTTGCTGATGATGCAATAACTAGAAGTTAACGAAAGTGGTAAAGTTTTATTTGAGTCCTTTGAAGGGGTTTCTACTGGAATATCATAGAAGTTCTTCTTTTCTACAGATGAAAGGATAGTAAGGTAATGCGGAGAAATATTCAACGAAGTATAAGAATCATATTGATATACAGTTTATTGATAGTTGGATTTTAATCGTGTGGCGGATATTTTCATTAACACAAAACAGATGCTGAAGCTCAAGTAGCCGTTCCATCTGGGACATAGTTAATCCGCTTCTGCCACCGATTCTCGACACCACCAACATCGGGTATGAATCTGTGACTCTGATGTTACAGAGCTTAATCTTGTGTACCAAAAACCACCATACCAACGGTGGTTTTCTCTGAGCTACTGCTCTTTGAGCCGAGGTAACTGGCTTGGAGGAGCTCAGTCACCAAAACTTGTTCTTTCAGTTTAGCCTTAACAGGCGCACAACTTAAGATGCTCCTCTAAATCAGTTACCAGTGGCTGCTGCCAGTGGCGCTTTGTCGTGCCTTCTGGGTTGGACTCAAGACGATAGTTACCGGGGCAGGCGCAGCGGTCGGACTGAACGGGGTTCGTGCGCTGGTCAGCTTGGGCGAACTGCCTGCGGAACGCTGAGTGTCAGGCGATGAGTAAAGTTTAAACAGCCATAACAGCAGGTGAAGAGCACCCCGGTGCCAAACCGAAGGCAGGAACAGAGCGCAGGGAGCTACCGGGGAAACGCAGGATCTTTATAGTCCTGTCCGGGTTTCGCCACCATGATTTGAGCGTCAGATTCTGTGATGCTTGTCAGGGGGCGGAGCCTATCGGAAAACCGGCAACGCGGCCTTTCTTGTTGCTTCTCCCATTCTCTCATTGCCTATGGCTTAATGTCTCTGTTCCCTCCGCTCGCCGCAGCGAACGACCGAGCGGGCGAGTCAGTGAGCGGAAGCGGAATATCTGCGGGCTTCTCTTTGGCACCGTACGCCATAGCGCATTTTAATACGATGCAGAATAGGGCGGGTACGCCGCAAAGTGACGTCACCTGACGTTCTGGATTACAAAGGTTAAAGCAGCGGACAGGAAATGTTTTGTGCCTAGCTATGCTATTCACAAGTAGCAGGACAGATGTGTTTTGGAGTACCATGACATAAAGACTCTGGTTAGCTCCCGCATGTTTGCTATCTTTGCTACCATTTGCTACTTTTGCTATCTCCAGGGTCTTCTGGTCTTTCAGTTGCTATCCTTTTGCTACTTTTGCTATCAAAATGCTACCTCTCCCTTCTTGCAATAAATGACCAAGGCACGTTAGCGATGTATACATCGCGCACCATAAGTGCCCCTTCCGAATCCCGTTCATCACTGTGTGATTAATCGAGGTTAAATCGACTACCAACGTAAAATCCTGCCATGCCTGACGGCTGACAACGCCCTCACGGTCGCGCAGTAAACTGCGCGCCCCTTTCACCGTTATCGGTGGGGTTCGTGGTGGCTTTCAGGGCGCTCGCCGGATTTTAACCGCTAAAATGAGCGATCCATGCGTTCGTG	LN:i:4357
```

Comment:
This sequence seems to be too long to me, maybe you can trim it - just for this tutorial?